### PR TITLE
View log in, sign up, forgot password and change password with i18n

### PR DESCRIPTION
**Pull request by Ukhanskyi**
[Trello Task](https://trello.com/c/kPjBf0z6/54-view-login-and-signup-pages)

**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

image

image

image

image

image

image

image

image

### CODE REVIEW

File: config/locales/ua.yml [hunk 1]
```diff
@@ -65,17 +65,44 @@ ua:
...
     password:
       new:
+        new_tab_title: 'Забули пароль?'
         new_title: 'Забули Ваш пароль?'
-        email_label: 'Електронна адреса: '
         email_placeholder: 'Введіть електронну адресу'
         submit_forgot_password: 'Надіслати інструкцію відновлення пароля'
 
       edit:
+        edit_tab_title: 'Зінити пароль'
```
Comment (mmarusyk) [inline config/locales/ua.yml:74]: orphographic error

**Review by mmarusyk — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/new.html.slim [hunk 2]
```diff
@@ -1,81 +1,63 @@
...
+          = title t('devise.registrations.new.tab_title'), t('devise.registrations.new.registration_title')
+        hr
+        = simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+          = f.error_notification
+          .container.w-100.justify-content-center
+            .field.mb-2
+              span.input-group.ml-2
+                i.fas.fa-user.mr-3.mt-3
+                .input-group-append
+                  = f.input :first_name, autofocus: true, placeholder: t('devise.registrations.new.f_name_placeholder'),
```
Comment (mmarusyk) [inline app/views/devise/registrations/new.html.slim:15]: Each argument from a new one

File: config/locales/ua.yml (hunk 1 above)
Comment (Ukhanskyi) [reply config/locales/ua.yml:74]: o-o-ops. thanks )

File: app/views/devise/registrations/new.html.slim (hunk 2 above)
Comment (Ukhanskyi) [reply app/views/devise/registrations/new.html.slim:15]: done

**Review by mmarusyk — APPROVED** [review]

File: Gemfile [hunk 3]
```diff
@@ -19,6 +19,7 @@ gem 'webpacker', '~> 4.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'awesome_print', '~> 1.8.0'
```
Comment (vikdotdev) [inline Gemfile:22]: це десь юзається?

Comment (vikdotdev) [reply Gemfile:22]: навіть якшо юзається то точно не в загальній групі воно повинно бути

Comment (vikdotdev) [reply Gemfile:22]: і компенти постав куди треба, або повидаляй, бо  `gem 'awesome_print', '~> 1.8.0'` тепер має комент `# Build JSON APIs with ease. `

Comment (Ukhanskyi) [reply Gemfile:22]: Це дуже чоткий гемчик, для читабельного виводу з консолі. Типу ap User.last 
і виведе кожне значення з нового рядочка.
image

Comment (Ukhanskyi) [reply Gemfile:22]: Виправив

**Review by matsevich — APPROVED** [review]

File: app/views/devise/registrations/new.html.slim [hunk 4]
```diff
@@ -1,81 +1,74 @@
...
+                  i.fas.fa-envelope.fa-lg.fa-fw.mt-2.control-label.col-sm-2.icon-prop
+                  .col-sm-10
+                    = f.input :email, autofocus: true, label: false,
+                      placeholder: t('devise.registrations.new.email_placeholder'),
+                      input_html: { class: 'form-control input-prop' }
+              p.form-group
+                .row
+                  i.fas.fa-venus-mars.fa-lg.fa-fw.mt-2.control-label.col-sm-2.icon-prop
+                  .col-sm-10
+                    = f.input :sex, collection: [[t('devise.registrations.new.sex.male'), 'male'],
```
Comment (pereslop) [inline app/views/devise/registrations/new.html.slim:38]: there is a helper which creates options, please look at UsersHelper

File: config/locales/en.yml [hunk 5]
```diff
@@ -57,16 +57,43 @@ en:
...
         submit_change_password: 'Change password'
 
+    registrations:
+      new:
+        tab_title: 'Sign up'
+        registration_title: 'Register Page'
+        f_name_placeholder: 'First name'
+        l_name_placeholder: 'Last name'
+        email_placeholder: 'Email'
+        sex:
```
Comment (pereslop) [inline config/locales/en.yml:78]: you don't need this tranlations, they at least exists in this file,

File: config/locales/ua.yml [hunk 6]
```diff
@@ -65,17 +65,44 @@ ua:
...
         submit_change_password: 'Змінити пароль'
 
+    registrations:
+      new:
+        tab_title: 'Реєстрація'
+        registration_title: 'Сторінка реєстрації'
+        f_name_placeholder: "Ім'я"
+        l_name_placeholder: 'Прізвище'
+        email_placeholder: 'Електронна скринька'
+        sex:
```
Comment (pereslop) [inline config/locales/ua.yml:87]: the same that is en.yml

File: config/locales/en.yml (hunk 5 above)
Comment (Ukhanskyi) [reply config/locales/en.yml:78]: Thanks, fixed!

File: app/views/devise/registrations/new.html.slim (hunk 4 above)
Comment (Ukhanskyi) [reply app/views/devise/registrations/new.html.slim:38]: Ohhh, I'm realy forgot about UserHelper

File: config/locales/ua.yml (hunk 6 above)
Comment (Ukhanskyi) [reply config/locales/ua.yml:87]: Ok, done.

**Pull request merged**